package.json

```diff
@@ -30,14 +30,15 @@
   },
   "homepage": "https://github.com/mixmaxhq/eslint-config-mixmax#readme",
   "optionalDependencies": {
-    "babel-eslint": "^7.2.3",
+    "babel-eslint": "^8.2.1",
+    "eslint-plugin-flowtype": "^2.41.0",
     "eslint-plugin-react": "^7.1.0"
   },
   "peerDependencies": {
-    "eslint": ">= 3"
+    "eslint": ">=4.14.0"
```

Comment:
can you please update the README with updated dependency installation instructions? (describing the optional `flowtype` dependency and updating the version of eslint that needs to be installed)

separately, it looks like the README is outdated in other ways - mentioning forthcoming configs that have already been added, not including the `jest` config, requiring node >=7.6.0, and describing the project as being at `v0.1` - i'm happy to clean these up after you merge this or feel free if you want to knock them out while you're in here.